mssqlcli/main.py

```diff
@@ -98,6 +98,7 @@
 class MssqlCli(object):
 
     max_len_prompt = 30
+    default_prompt = '\\d> '

```

Comment:
I don't think a trailing space is a common convention. Can we remove the space after the '>' for the default?

Reply:
I think it is. Both mycli and pgcli use trailing ` ` in their prompt:

https://github.com/dbcli/pgcli/blob/327f15afc3fcbd556b87a5b568b2b6c983519bdd/pgcli/pgclirc#L123

https://github.com/dbcli/mycli/blob/d6ccd390a9b4b659a2bd35e8bf94172eb9c3874e/mycli/myclirc#L67

Without the ` `, once you start typing, it looks like the prompt and the query are glued together:

```
foo>select 1
```

not pretty!

```
foo> select 1
```

pretty!

Reply:
Fair enough. I don't feel strongly either way, so a space it is.